tests/UI/pages/BO/quickAccess/index.js

```diff
@@ -62,6 +63,15 @@ class QuickAccess extends BOBasePage {
     await this.clickAndWaitForNavigation(page, this.addNewQuickAccessButton);
   }
 
+  /**
+   * Remove the quick access for current page
+   * @param page {Page} Browser tab
+   * @returns {Promise<void>}
+   */
+  async removeQuickAccessPage(page) {
+    await this.clickAndWaitForNavigation(page, this.removeQuickAccessLink);
+  }
+  
```

Comment:
Why did you create this method ? 
I think you didn't use it in your test

Reply:
I think it's used on the post test [here](https://github.com/PrestaShop/PrestaShop/pull/30049/files#diff-ddf82f3956e39108457032f7d8bc280065a08e1dda61989f64127e0fa7351f2dR97)

Reply:
`await countriesPage.removeLinkFromQuickAccess(page);`
It's not the same method :/

Reply:
you're right, `removeLinkFromQuickAccess` is from the `BoBasePage` so no need for this one probably.

Reply:
That's true. I fixed it